views/groups.handlebars

```diff
@@ -1,20 +1,24 @@
-<body>
```

Comment:
I still have more to do with the navbar but I will try to keep any id tags the same.